lib/tasks/webpacker.rake

```diff
@@ -1,13 +1,13 @@
 tasks = {
-  'webpacker:install' => 'Installs and setup webpack with yarn',
-  'webpacker:compile' => 'Compiles webpack bundles based on environment',
-  'webpacker:install:react' => 'Installs and setup example react component',
-  'webpacker:install:vue' => 'Installs and setup example vue component',
-  'webpacker:install:angular' => 'Installs and setup example angular2 component'
+  "webpacker:install"         => "Installs and setup webpack with yarn",
+  "webpacker:compile"         => "Compiles webpack bundles based on environment",
+  "webpacker:install:react"   => "Installs and setup example react component",
+  "webpacker:install:vue"     => "Installs and setup example vue component",
+  "webpacker:install:angular" => "Installs and setup example angular2 component"
 }.freeze
 
-desc 'Lists available tasks under webpacker'
+desc "Lists available tasks under webpacker"
 task :webpacker do
-  puts 'Available webpacker tasks are:'
+  puts "Available webpacker tasks are:"
```

Comment:
That's fine. I don't care too much. I just remember this comment but I'm probably mixing different style guides for various languages and projects I work on.

> Style: Use '' for keys etc without interpolation. Use "" for sentences, prose, or for interpolation